spec/index.html

```diff
@@ -342,7 +342,7 @@ <h2>Introduction</h2>
 
     <div class="ednote">
       See YAML-LD description of this spec at
-      <a href="https://github.com/json-ld/yaml-ld/blob/main/spec/spec.yaml">
+      <a href="https://github.com/json-ld/yaml-ld/blob/main/spec/data/spec.yaml" target="_blank">
```

Comment:
It's cleaner to keep these references as relative to the spec, so when they're published, they show up properly. That would make it `<a href="data/spec.yaml">...</a>`.